README.md

```diff
@@ -115,7 +115,7 @@ You should now see a generated file at `src/gen/example_pb.ts` that contains a c
 
 * [connect-es](https://github.com/bufbuild/connect-es):
   Type-safe APIs with Protobuf and TypeScript.
-* [connect-es-integration](https://github.com/bufbuild/connect-es-integration):
+* [examples-es](https://github.com/connectrpc/examples-es):
```

Comment:
Let's not use just the repository name here, it's misleading without the org name.

```suggestion
* [connect-es Examples](https://github.com/connectrpc/examples-es):
```